public/modules/custom/helfi_hakuvahti/helfi_hakuvahti.install

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @file
+ * Contains install functions for HELfi hakuvahti -module.
+ */
+
+declare(strict_types=1);
+
+use Drupal\Core\Database\Database;
+
+/**
+ * The schema.
+ */
+function helfi_hakuvahti_schema() {
+  $schema['hakuvahti_selected_filters'] = [
+    'description' => 'The table for storing selected hakuvahti filters.',
+    'fields' => [
+      'id'  => [
+        'type' => 'serial',
+        'not null' => TRUE,
+      ],
+      // Can be used to get the count of created hakuvahti-subscriptions.
+      'token' => [
+        'type' => 'varchar',
+        'length' => 255,
+        'not null' => TRUE,
+        'default' => '',
+      ],
+      'filter_name' => [
+        'description' => 'The name of the filter.',
+        'type' => 'varchar',
+        'length' => 255,
+        'not null' => TRUE,
+        'default' => '',
+      ],
+      'filter_value' => [
+        'description' => 'Selected filter value.',
+        'type' => 'varchar',
+        'length' => 255,
```

Comment:
This should be longer than 255. Maybe 1024 or even 2048 characters(?)

Reply:
Should not